frontend/src/components/ImplementationReportView/ImplementationReportView.tsx

```diff
@@ -121,6 +121,22 @@ const ReportComponent: React.FC<{ implementation: Implementation }> = ({
                   </ul>
                 </td>
               </tr>
+              {implementation.links && !!implementation.links.length && (
```

Comment:
I think this potentially can be shortened to `implementation.links?.length`

Reply:
I agree.

Reply:
@harrel56, kindly review.